WBooks/App.js

```diff
@@ -9,27 +9,70 @@ import 'react-native-gesture-handler';
 import React, { Component } from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
-import { StatusBar } from 'react-native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { StyleSheet, StatusBar, Image } from 'react-native';
 
 import BookList from './src/app/screens/Library/screens/BookList';
 import BookDetails from './src/app/screens/BookDetails/screens/BookDetails';
+import Wishlist from './src/app/screens/Wishlist';
 import CustomHeader from './src/app/components/CustomHeader';
-import { white } from './src/constants/colors';
+import { white, blue, lightGray } from './src/constants/colors';
+import images from './src/constants/images';
+import { library, details, wishlist } from './src/constants/routes';
 
-const Stack = createStackNavigator();
+const Tab = createBottomTabNavigator();
+const LibraryStack = createStackNavigator();
+
+const styles = StyleSheet.create({
+  tabBarItem: {
+    width: 25,
+    height: 25
+  }
+});
 
 class App extends Component {
   customHeader = () => <CustomHeader />;
 
+  customTabBarIcon = ({ icon }) => <Image source={icon} style={styles.tabBarItem} />;
+
+  LibraryStackScreen = () => (
+    <LibraryStack.Navigator screenOptions={{ headerBackground: this.customHeader, headerTintColor: white }}>
+      <LibraryStack.Screen name={library} component={BookList} />
+      <LibraryStack.Screen name={details} component={BookDetails} />
+    </LibraryStack.Navigator>
+  );
+
   render() {
     return (
       <>
         <NavigationContainer>
           <StatusBar barStyle="light-content" />
-          <Stack.Navigator screenOptions={{ headerBackground: this.customHeader, headerTintColor: white }}>
-            <Stack.Screen name="Library" component={BookList} />
-            <Stack.Screen name="Details" component={BookDetails} />
-          </Stack.Navigator>
+          <Tab.Navigator
```

Comment:
okay, now this `App` component is too big so is a good idea to move all about navigation to a separated file like `navigation.js` it can be on a config folder.
Then just import here you `Navigation` component to render.

Reply:
LibraryStackScreen, customTabBarIcon and  customHeader too. all into the same file